appveyor.yml

```diff
@@ -21,7 +21,8 @@ services:
 
 before_build:
   # Install dotnet cli
-  - ps: iex ((New-Object Net.WebClient).DownloadString('https://raw.githubusercontent.com/Nick-Lucas/cli/91a985ebd0a70c5150135db0a15223f229cb1a34/scripts/obtain/dotnet-install.ps1'))
+  - appveyor DownloadFile https://raw.githubusercontent.com/dotnet/cli/v1.0.0-preview2/scripts/obtain/dotnet-install.ps1
```

Comment:
Does this also resolve the issue with 'Exit 0' in the installer causing the entire build process to exit?
I've been looking into ways around that so we can move back to the official script